fhir-smart/src/test/java/org/linuxforhealth/fhir/smart/test/AuthzPolicyEnforcementTest.java

```diff
@@ -353,6 +353,39 @@ public void testDirectPatientInteraction() throws FHIRPersistenceInterceptorExce
         }
     }
 
+    @Test
+    public void testUserInteraction() throws FHIRPersistenceInterceptorException {
```

Comment:
This claims to throw FHIRPersistenceInterceptorException but it catches it internally

Reply:
good catch
```suggestion
    public void testUserInteraction() {
```